### ISSUE TITLE
flathub verification?

### ISSUE THREAD

**Issue by qoijjj** [flathub/org.videolan.VLC#197]
Getting verification would allow an increased level of user trust.

https://discourse.flathub.org/t/how-do-you-get-a-flathub-submission-verified/3714
https://flathub.org/apps/collection/verified/1

I believe this would require the blessing of upstream devs.

**Comment by AsciiWolf** [comment]
Linux Mint [will hide](https://www.omgubuntu.co.uk/2024/06/linux-mint-will-hide-unverified-flatpaks) unverified Flatpak apps from search results (and other distributions will probably follow). Please, consider verifying the VLC Flatpak.

**Comment by MathieuBordere** [comment]
bump :-)

**Comment by Erick555** [comment]
I don't think this flatpak is officially endorsed by vlc team: https://code.videolan.org/videolan/vlc/-/issues/28356

**Comment by nolddor** [comment]
I am not sure if this is possible, but I would like to see a collaboration between the maintainer of current flatpak build and VLC Team to deliver a verified version of the software.

I think they are open to do so, see https://code.videolan.org/videolan/vlc/-/issues/28356
This will elevate trust among end consumers of VLC and Flatpak.

@MatMaul do you see any possibility or potential talks with them?
Most OSes built-in stores hide non-verified flatpaks from their searches, which is another thing to take into consideration.

Best Regards

**Comment by MatMaul** [comment]
Thanks all for pushing for this.

I answered on VLC bugtracker:

https://code.videolan.org/videolan/vlc/-/issues/28356#note_481649

**Comment by Perkolator** [comment]
@MatMaul I don't know how you find the strength to do this with those irritating vlc devs, they are the worst devs I have ever seen (not based on just this issue but many others... arrogant, offensive, condescending (even when shown to be wrong about something)...).

BTW. I just noticed that version 3.0.22 was never pushed as a flatpak package here, how come? Also, the automatic PR #284 from flathubbot failed for 3.0.23 four days ago.
